crates/holo_hash/src/hash.rs

```diff
@@ -99,7 +99,7 @@ where
         use proptest::strategy::Strategy;
 
         let strat = T::arbitrary().prop_flat_map(move |hash_type| {
-            let gen_strat = proptest::string::bytes_regex(r".[39]").unwrap();
+            let gen_strat = proptest::string::bytes_regex(r"(?-u:.{39})").unwrap();
```

Comment:
Might be worth a comment? I have no idea what the regex for `?-u:` means. Also... shouldn't this be only valid base64 characters?

Reply:
The docs for `bytes_regex` explains the syntax but I can add a comment too.

No, the 39 bytes are free to be anything, since they're based on a hash.